onnx/defs/operator_sets.h

```diff
@@ -1164,4 +1164,10 @@ inline void RegisterOnnxOperatorSetSchema(int target_version) {
   OpSchemaRegistry::Instance()->SetLoadedSchemaVersion(target_version);
 }
 
+inline void DeregisterOnnxOperatorSetSchema() {
```

Comment:
Since we're manipulating the static map `OpName_Domain_Version_Schema_Map` at runtime, how should we best guarantee thread-safety here?